### DISCUSSION TITLE
Linux support?

### DISCUSSION THREAD

**Discussion by blodwehd** [rocksdanister/lively#1178] [Q&A]
Any idea if this works with Linux under WINE, or if it can be jerry-rigged to work natively? Any plans to port it in the future?

**Comment by rocksdanister** [comment]
>Any idea if this works with Linux under WINE

Rewriting the UI program is not a problem, I already have a basic UI written without using WINE:
https://github.com/rocksdanister/lively-linux

>if it can be jerry-rigged to work natively? Any plans to port it in the future?

Lively is a system application and each operating system will require its own implementation https://github.com/rocksdanister/lively/issues/220 
Linux has many desktop managers so this is quite hard and I am new to linux system programming so help is needed.

**Reply by ghost** [reply]
I wish I could help… I really do… what if you could sync wallpapers between versions, in a dual boot? Idk if this is even theoretically possible but if it is, it'd be super cool

**Reply by ghost** [reply]
Like one is win10 and one is Arch Linux (kde)

**Reply by Sounak008** [reply]
> > Any idea if this works with Linux under WINE
> 
> Rewriting the UI program is not a problem, I already have a basic UI written without using WINE: https://github.com/rocksdanister/lively-linux
> 
> > if it can be jerry-rigged to work natively? Any plans to port it in the future?
> 
> Lively is a system application and each operating system will require its own implementation #220 Linux has many desktop managers so this is quite hard and I am new to linux system programming so help is needed.

Maybe make a daemon like swww does ?
